examples/oauth2/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
+services:
+  gateway:
+    image: apicast-test:latest
```

Comment:
shouldn't this point to `quay.io/3scale/apicast:v2` ? people will not have `apicst-test` unless they built it locally